bdleon/nashville-sell-and-trade
# added all models needed

#### Changes Made
1. Added file `category.py` to `models` directory.
1. Added file `message.py` to `models` directory.
1. Added file `nash_user.py` to `models` directory.
1. Added file `product_category.py` to `models` directory.
1. Added file `product.py` to `models` directory.
1. Added file `save_post.py` to `models` directory.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bd-models
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > check database to see if models match what is being presented in Nashville S and T DB
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
